src/components/SearchableMultiSelect/SearchableMultiSelect.tsx

```diff
@@ -423,22 +561,22 @@ const SearchableMultiSelect = createClass({
 			>
 				<CheckboxLabeled
 					className={cx('&-CheckboxLabeled')}
-					callbackId={optionIndex}
+					callbackId={optionIndex.toString()}
 					isSelected={_.includes(selectedIndices, optionIndex)}
 				>
 					<CheckboxLabeled.Label>
 						{_.isString(optionProps.children)
 							? this.renderUnderlinedChildren(optionProps.children, searchText)
 							: _.isFunction(optionProps.children)
-								? React.createElement(optionProps.children, { searchText })
-								: optionProps.children}
+							? React.createElement(optionProps.children, { searchText })
+							: optionProps.children}
 					</CheckboxLabeled.Label>
 				</CheckboxLabeled>
 			</DropMenu.Option>
 		);
-	},
+	};
 
-	renderOptions() {
+	renderOptions = (): any => {
```

Comment:
Same as `renderOption`, this could have a stricter return type.

Reply:
fixed